spec/requests/memberships_controller_spec.rb

```diff
@@ -38,9 +38,10 @@
 
   describe "PUT #update" do
     let(:membership) { create(:membership, user:, group:) }
+    let(:logged_in_user) { organisation_admin_user }
```

Comment:
I think we call this `current_user` in other tests, might be worth using the same here for consistency?

Reply:
yes, logged_in_user is a clunky name, I'll change it.

Reply:
I've changed it 👍